specification/search/data-plane/Azure.Search/preview/2019-05-06-preview/searchservice.json

```diff
@@ -5771,6 +5771,71 @@
           "default": 0,
           "description": "The maximum number of items in a single batch that can fail indexing for the batch to still be considered successful. -1 means no limit. Default is 0."
         },
+        "parsingMode": {
```

Comment:
Shouldn't all these be defined as properties of "configuration"? As authored, these are sibling to "configuration" and is probably why the breaking change error is ocurring.

Reply:
Yes, I think these should be under configuration (which still needs to have additionalProperties in case we add more parameters over time).

Reply:
Fixed in commit dfe05dffdc66c97a9d0229a9fe34456fb73f7444.